packages/app-builder-lib/src/options/SquirrelWindowsOptions.ts

```diff
@@ -40,6 +40,11 @@ export interface SquirrelWindowsOptions extends TargetSpecificOptions {
    */
   readonly useAppIdAsId?: boolean
 
+  /**
+   *  The custom squirrel.windows.exe path. Defaults to `https://github.com/electron-userland/electron-builder-binaries/tree/master/Squirrel.Windows`.
+   */
+  readonly vendorDirectory?: string
+
```

Comment:
Please update doc comment
```
/**
*  The custom file path to `squirrel.windows.exe` for usage with [electron-winstaller](https://github.com/electron/windows-installer) `vendorDirectory`.
*/
```

>Defaults to `https://github.com/electron-userland/electron-builder-binaries/tree/master/Squirrel.Windows`

Also, is this true with the current setup? I don't see any default value being set for `vendorDirectory`. As this is currently coded, it looks like it'll default to the vendor files that come with `electron/windows-installer`

Reply:
updated.

/**
   * The custom squirrel.windows.exe path. If not specified will use the Squirrel.Windows that is shipped with electron-installer(https://github.com/electron/windows-installer/tree/main/vendor).
   * After https://github.com/electron-userland/electron-builder-binaries/pull/56 merged, will add `electron-builder-binaries` to get the latest version of squirrel.
   */
 
 The squirrel windows in this  https://github.com/electron-userland/electron-builder-binaries/pull/56 is the same as the one in https://github.com/electron/windows-installer/tree/main/vendor, so when this value is empty, it defaults to using the squirrel windows from windows-installer.